cosign.pub

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEQ8hMf0oDYyKmMQZdtDEZs6iTJWvX
+r4lOvDAH5xnnL2w48oiGmt012qwUfvUIBAMViIjByJ+3g6QzKpvEfZl1vQ==
+-----END PUBLIC KEY-----
```

Comment:
Looks like this is the key that is used to sign a couple of images that are hosted in dockerhub. Do I have that right?

Reply:
This is the public key to verify those images